docs/src/reference.rst.template

```diff
@@ -16,3 +16,4 @@ Reference
 
 .. toctree::
    :maxdepth: 1
+
```

Comment:
Since this is so sensitive, can we add a comment line to mention the blank line MUST REMAIN?

Reply:
Happy to do that in a later commit. I want to get this release out ASAP